content/kubeone/master/guides/local/_index.en.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Local subcommand"
+date = 2022-08-05T15:58:00+02:00
+enableToc = true
++++
+
+## Introduction
+
+`kubeone local` subcommands helps you to initialize a **local** (e.g. on the
+local machine, in the current shell), 1 node all-in-one cluster.
```

Comment:
It would be nice to clarify what exactly all-in-one means.

Reply:
What else besides "1 node all-in-one cluster" can it mean?!

Reply:
I thought about clarifying that it's both a control plane and a worker node. Something like:

> 1 node all-in-one cluster (the node is running both control plane and workloads)

Reply:
Fixed